src/Stratis.Bitcoin.IntegrationTests/Connectivity/ConnectivityTests.cs

```diff
@@ -26,47 +26,6 @@ public ConnectivityTests()
             this.powNetwork = new BitcoinRegTest();
         }
 
-        /// <summary>
-        /// Peer A_1 connects to Peer A_2
-        /// Peer B_1 connects to Peer B_2
-        /// Peer A_1 connects to Peer B_1
-        ///
-        /// Peer A_1 asks Peer B_1 for its addresses and gets Peer B_2
-        /// Peer A_1 now also connects to Peer B_2
-        /// </summary>
-        [Fact]
-        [Trait("Unstable", "True")]
-        public void Ensure_Peer_CanDiscover_Address_From_ConnectedPeers_And_Connect_ToThem()
```

Comment:
This is OK to be removed as we know that `PeerDiscovery` works.

Reply:
We know it works is the reason why we should have a test in there. So that we know it still works.